### PR TITLE
Add client for  Certificate Authority

### PR DESCRIPTION
**Pull request by balestrinc**

### CODE REVIEW

**Comment by marikrg** [comment]
> I'm not sure about the class name.
> Maybe certificate_authority could be a better way, since that there are others certificates, like rabbitmq.

I agree.
For me, beyond the class name, the modules name also should be more specific.

File: hpOneView/resources/security/certificates.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        then the internal CA root certificate, RabbitMQ client certificate and RabbitMQ server certificate will be
+        regenerated. This will invalidate the previous version of RabbitMQ client certificate and the RabbitMQ server
+        will be restarted to read the latest certificates.
+
+        Args:
+            alias_name (str): Alias name.
+            timeout:
+                Timeout in seconds. Wait for task completion by default. The timeout does not abort the operation
+                in OneView, just stop waiting for its completion.
+        """
```
Comment (marikrg) [inline hpOneView/resources/security/certificates.py]: The `Returns` is missing in the docstring.

File: hpOneView/resources/security/certificates.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+class Certificates(object):
+    """
+    Certificates API client.
+    """
+
+    URI = '/rest/certificates/ca'
+
+    def __init__(self, con):
+        self._connection = con
```
Comment (marikrg) [inline hpOneView/resources/security/certificates.py]: Looks like this instance variable is not being used.

**Comment by GustavoHennig** [comment]
The first class name that comes to my mind is: CertificatesClientRabbitMq

I think I would do the same thing to the module name: certificates_client_rabbitmq

We have conflict names with: 
- Certificate Authority
- Certificates Client RabbitMq
- Client Certificates

Another option is to put all these resources together.

**Pull request merged**